src/App.tsx

```diff
@@ -11,11 +11,15 @@ export class App extends React.Component<{}, State> {
   };
 
   addOne = () => {
-    // write code here
+    this.setState(state => ({
+      count: state.count + 1,
+    }));
```

Comment:
You can use destructurization to cut down your code
```suggestion
    this.setState(({ count })=> ({
      count: count + 1,
    }));
```